apps/frontend/src/app/3_organisms/RewardHistory/RewardHistory.tsx

```diff
@@ -0,0 +1,215 @@
+import { ExportCSV } from '../../2_molecules/ExportCSV/ExportCSV';
+import { TransactionTypeRenderer } from '../../2_molecules/TransactionTypeRenderer/TransactionTypeRenderer';
+import { useNotificationContext } from '../../../contexts/NotificationContext';
+import { useAccount } from '../../../hooks/useAccount';
+import { translations } from '../../../locales/i18n';
+import { zeroClient } from '../../../utils/clients';
+import { Bitcoin, EXPORT_RECORD_LIMIT } from '../../../utils/constants';
+import {
+  StabilityDepositChange,
+  StabilityDepositChange_Filter,
+  useGetStabilityDepositChangesLazyQuery,
+} from '../../../utils/graphql/zero/generated';
+import { dateFormat } from '../../../utils/helpers';
+import { formatValue } from '../../../utils/math';
+import { useGetRewardHistory } from './hooks/useGetRewardHistory';
+
+const DEFAULT_PAGE_SIZE = 10;
+
+export const RewardHistory: FC = () => {
+  const { t } = useTranslation();
+  const { account } = useAccount();
+  const { addNotification } = useNotificationContext();
+
+  const [page, setPage] = useState(0);
+  const [pageSize, setPageSize] = useState(DEFAULT_PAGE_SIZE);
+  const chain = chains.find(chain => chain.id === defaultChainId);
+
+  const [orderOptions, setOrderOptions] = useState<OrderOptions>({
+    orderBy: 'sequenceNumber',
+    orderDirection: OrderDirection.Desc,
+  });
+
+  const { data, loading } = useGetRewardHistory(
+    account,
+    pageSize,
+    page,
+    orderOptions,
+  );
+  const [getStabilityDeposit] = useGetStabilityDepositChangesLazyQuery({
+    client: zeroClient,
+  });
+
+  const renderCollateralChange = useCallback((collateralGain: string) => {
+    return `${formatValue(
+      Math.abs(Number(collateralGain)),
+      8,
+    )} ${SupportedTokens.rbtc.toUpperCase()}`;
+  }, []);
+
+  const generateRowTitle = useCallback((row: StabilityDepositChange) => {
+    return (
+      <Paragraph size={ParagraphSize.small}>
+        <TransactionTypeRenderer type={row.stabilityDepositOperation} />
+        {' - '}
+        {dateFormat(row.transaction.timestamp)}
+      </Paragraph>
+    );
+  }, []);
+
+  const columns = useMemo(
+    () => [
+      {
+        id: 'sequenceNumber',
+        title: t(translations.rewardHistoryTable.table.timestamp),
+        cellRenderer: (tx: StabilityDepositChange) =>
+          dateFormat(tx.transaction.timestamp),
+        sortable: true,
+      },
+      {
+        id: 'stabilityDepositOperation',
+        title: t(translations.rewardHistoryTable.table.transactionType),
+        cellRenderer: (tx: StabilityDepositChange) => (
+          <TransactionTypeRenderer type={tx.stabilityDepositOperation} />
+        ),
+      },
+      {
+        id: 'collateralGain',
+        title: t(translations.rewardHistoryTable.table.rewardChange),
+        cellRenderer: tx => (
+          <AmountRenderer
+            value={tx.collateralGain || 0}
+            sufix={Bitcoin}
+            {...applyDataAttr('reward-history-collateral-gain')}
```

Comment:
```suggestion
            dataAttribute="reward-history-collateral-gain"
```
This won't work as `AmountRenderer` is a component and not a native HTML element.